content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -148,6 +148,7 @@ different Kubernetes components.
 | `ServiceNodeExclusion` | `false` | Alpha | 1.8 | 1.18 |
 | `ServiceNodeExclusion` | `true` | Beta | 1.19 | |
 | `ServiceTopology` | `false` | Alpha | 1.17 | |
+| `SizeMemoryBackedVolumes` | `false` | Alpha | 1.20 | |
```

Comment:
Just need a short description below describing this feature. Then, I think docs are good to merge